packages/kit/content/docs/basics.md

```
@@ -124,6 +124,31 @@ If you `await` every load inside `init()`, the loop has not started yet, so ther
 for small games. Per-sheet hot-reload replacements also expose `sheet.status` (`'loading' | 'ready' | 'failed'`) and a
 coarse `sheet.progress` (`0` or `1.0`). Prefer `BT.loadingAssetsCount` for one engine-wide signal.
 
+## The splash
+
+A short BLIT386 splash plays before your game starts, but only in a real build. Running `npx blit run` (or
+`npm run dev`) is a development build, so you will not see it while you work. To check it, build and preview:
+
+    npm run build
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)